src/i18n/i18n.translations.ts

```diff
@@ -0,0 +1,49 @@
+import { namespaces } from "./i18n.constants";
+
+export const en = {
+    [namespaces.common]: {
+      buttons: {
+        ok: "Ok",
+        cancel: "Cancel",
+      },
+    },
+    [namespaces.pages.hello]: {
+      welcome: "Welcome to Cultural Survival.",
+      categories: "Categories",
+      radio: "Radio",
+      grants: "Grants",
+      
+    },
+    [namespaces.pages.grants]: {
+      grants_feed: "Grants Available",
+      
+    },
+    [namespaces.pages.audio]: {
+      audio_feed: "AudioFEED_Feed",
+      
+    },
+  };
+
+export const es = {
+  [namespaces.common]: {
+    buttons: {
+      ok: "Aceptar",
+      cancel: "Cancelar",
+    },
+  },
+  [namespaces.pages.hello]: {
+    welcome: "Bienvenida a Cultural Survival.",
```

Comment:
Should be "bienvenidos" -- bienvenida is for "... is welcome", bienvenidos is for "welcome to ..."